asynq/futures.py

```diff
@@ -126,7 +126,13 @@ def _computed(self):
         # the notification even in case one of them fails.
         # Otherwise lots of negative effects are possible - e.g.
         # some of them might be left in blocked state.
-        self.on_computed.safe_trigger(self)
+        try:
+            self.on_computed.safe_trigger(self)
+
+        # pass on any exception in the callback, since we don't expect the caller to know how to
+        # deal with it
+        except BaseException:
```

Comment:
This may end up eating a KeyboardInterrupt. What happens if we only catch Exceptions here? Even for those it would be preferable to log them somehow.